algorithm-3term/자연수-뒤집어-배열로-만들기.js

```diff
@@ -0,0 +1,7 @@
+function solution(n) {
+    const numToStr = Array.from(`${n}`).reverse();
```

Comment:
```suggestion
    const numToArray = Array.from(`${n}`).reverse();
```
주어진 숫자를 무자열로 변환한다는 뜻에서 `numToStr` 변수명을 사용한 것 같은데 변수 값을 보면 추가로 배열로 만들고 역순 정렬도 되어있습니다. 데이터 타입이 배열이므로 `배열`과 연관된 변수명을 사용하는 것이 좋을 것 같습니다. (ex : `numToArr`)